arviz/tests/base_tests/test_stats.py

```diff
@@ -47,6 +47,12 @@ def test_hpd():
     assert_array_almost_equal(interval, [-1.88, 1.88], 2)
 
 
+def test_hpd_multidimension():
```

Comment:
There are some more cases to test:

* test ndarray input:
   * 1d array -> done in `test_hpd`
   * 2d array -> missing, this is important to make sure that hpd with `wrap_apply_ufunc` is backwards compatible
   * 3d+ array -> done in this test
* test idata input
   * missing, the `centered_eight` fixture could be used for this. This should test returned shapes, returned types (xarray dataset) and that the `group` and `var_names` arguments work

Reply:
It would also be great to add a test with idata case using `kwargs` to get the hpd of every chain

Reply:
I have added all the specified tests, except for `group` one. Can you help me with this one?